src/components/Field.jsx

```diff
@@ -37,17 +39,24 @@ const Field = (props) => {
 export default Field
 
 export const FieldWrapper = ({ required, label, dirty, touched, errors, children }) => {
-  var classes = classNames('account-field', {
```

Comment:
Why not use `classNames` anymore? I think this new way more complicated but maybe there is a detail that I don't see here.